vignettes/vitorcid.Rmd

```diff
@@ -20,26 +20,66 @@ knitr::opts_chunk$set(
 library(vitorcid)
 ```
 
-# Overview
-To be added soon.
+# vitorcid
+
+The `vitorcid` package that allows you to generate a CV based on your ORCID profile.
+
+# Installation
+
```

Comment:
Provide information regarding installation here, sth like:

(...)
You can install the package using `remotes` in the following way:
``` 
remotes::install_github("https://github.com/gladkia/vitorcid.git", ref = "main")
```
(...)